tests/test_sourmash.py

```diff
@@ -2563,6 +2563,37 @@ def test_gather_metagenome():
                     'NC_011294.1 Salmonella enterica subsp...' in out))
 
 
+def test_gather_metagenome_num_results():
+    # set a threshold on the gather output
+    with utils.TempDirectory() as location:
```

Comment:
I think you could use the `@utils.in_tempdir` decorator for this test function. Actually, a question of mine is how important are python decorators? Do they help the code run, or are they really just for readability purposes.

Reply:
hah, good catch! thanks!

in this case, for readability purposes. but don't discount readability, it's one of the most important things in a code base!